cli/lsp/code_lens.rs

```diff
@@ -282,3 +501,80 @@ pub(crate) async fn tsc_code_lenses(
   });
   Ok(Rc::try_unwrap(code_lenses).unwrap().into_inner())
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
+  use crate::media_type::MediaType;
+
+  #[test]
+  fn test_deno_test_collector() {
+    let specifier = resolve_url("https://deno.land/x/mod.ts").unwrap();
+    let source = r#"
+      Deno.test({
+        name: "test a",
+        fn() {}
+      });
+
+      Deno.test("test b", function anotherTest() {});
+    "#;
```

Comment:
Could you squeeze in a test for:
```
const { test } = Deno;

test("test c", function yetAnotherTest() {});
```

Reply:
Nvm, I see there are other tests like this later in the PR

Reply:
Yeah, the integration test is comprehensive, the unit test is sort of basic.